### PR TITLE
Add documentation to recover CELO from Alfajores Testnet

### PR DESCRIPTION
**Pull request by henry-zhang-bohan**
Create documentation similar to https://docs.celo.org/celo-holder-guide/eth-recovery but for Alfajores wallet

### CODE REVIEW

File: packages/docs/SUMMARY.md [hunk 1]
```diff
@@ -26,6 +26,7 @@
 - [Voting on Validator Groups](celo-holder-guide/voting-validators.md)
 - [Voting on Governance](celo-holder-guide/voting-governance.md)
 - [Recovering CELO from an Ethereum Address](celo-holder-guide/eth-recovery.md)
+- [Recovering CELO from Alfajores Wallet](celo-holder-guide/alfajores-recovery.md)
```
Comment (marekolszewski) [inline packages/docs/SUMMARY.md]: Maybe we can create a folder (e.g. Recovering CELO from other wallets) that contains these two sections in it?

File: packages/docs/celo-holder-guide/alfajores-recovery.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+3. Replace the `<word>`s in brackets with the 24 words from your recovery phrase (mnemonic)
+4. Press ctrl-o to save
+5. Press ctrl-x to exit
+
+### Recover your Alfajores address on Celo
+
+Recover your Alfajores address on Celo Mainnet:
+
+```
+celocli account:recover-old --mnemonicPath recovery.txt --node https://rc1-forno.celo-testnet.org
```
Comment (marekolszewski) [inline packages/docs/celo-holder-guide/alfajores-recovery.md]: Might be worth chatting with Ponti about making this be a flag passed to account:new? e.g. --legacyWallet?

**Review by marekolszewski — CHANGES_REQUESTED** [review]
Looks good! Left some comments.

File: packages/docs/SUMMARY.md (hunk 1 above)
Comment (henry-zhang-bohan) [reply packages/docs/SUMMARY.md]: moved

File: packages/docs/celo-holder-guide/alfajores-recovery.md (hunk 2 above)
Comment (henry-zhang-bohan) [reply packages/docs/celo-holder-guide/alfajores-recovery.md]: will chat with Ponti, but I suspect it'll be possible to publish a new version this week

Comment (dominiquejane) [reply packages/docs/celo-holder-guide/alfajores-recovery.md]: @marekolszewski @gastonponti Is it ok to start pointing users to this document?

Comment (gastonponti) [reply packages/docs/celo-holder-guide/alfajores-recovery.md]: yes

**Review by barbaraliau — APPROVED** [review]

**Pull request closed without merging**